ansible/roles/cri/templates/containerd.conf.j2

```diff
@@ -0,0 +1,2 @@
+overlay
```

Comment:
Because there isn't really anything to template here I think this can be moved to the `file/` dir. See here for an example: https://github.com/raspbernetes/k8s-cluster-installation/tree/master/ansible/roles/common/files

Reply:
will do !